default.hbs

```diff
@@ -52,23 +52,7 @@
     height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
     <!-- End Google Tag Manager (noscript) -->
 
-    <div class="h-banner">
-        <div class="h-pad">
-            <div class="h-wrap">
-                <ul class="h-nav">
-                    <li class="tab-home"><a href="https://holochain.org/"><img src="{{asset "img/holochain-wordmark-logo-2x.png"}}" alt="Holochain.org" class="header-logo header-logo-large"><img src="{{asset "img/holochain-logo.png"}}" alt="Holochain.org" class="header-logo header-logo-small"></a></li>
-                    <li class="tab-developers"><a href="https://developer.holochain.org/">Developers</a></li>
-                    <li class="tab-discord"><a href="https://discord.gg/HhhxQkNUsr"><svg role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 127.14 96.36" style="height:1em"><path style="fill:currentColor" d="M107.7,8.07A105.15,105.15,0,0,0,81.47,0a72.06,72.06,0,0,0-3.36,6.83A97.68,97.68,0,0,0,49,6.83,72.37,72.37,0,0,0,45.64,0,105.89,105.89,0,0,0,19.39,8.09C2.79,32.65-1.71,56.6.54,80.21h0A105.73,105.73,0,0,0,32.71,96.36,77.7,77.7,0,0,0,39.6,85.25a68.42,68.42,0,0,1-10.85-5.18c.91-.66,1.8-1.34,2.66-2a75.57,75.57,0,0,0,64.32,0c.87.71,1.76,1.39,2.66,2a68.68,68.68,0,0,1-10.87,5.19,77,77,0,0,0,6.89,11.1A105.25,105.25,0,0,0,126.6,80.22h0C129.24,52.84,122.09,29.11,107.7,8.07ZM42.45,65.69C36.18,65.69,31,60,31,53s5-12.74,11.43-12.74S54,46,53.89,53,48.84,65.69,42.45,65.69Zm42.24,0C78.41,65.69,73.25,60,73.25,53s5-12.74,11.44-12.74S96.23,46,96.12,53,91.08,65.69,84.69,65.69Z"/></svg> Discord</a></li>
-                    <li class="tab-blog"><a href="{{@site.url}}">Blog</a></li>
-                    <li class="tab-events"><a href="https://holochain.org/events/">Events</a></li>
-                </ul>
-                <ul class="h-icons">
-                    <li><a href="{{@site.url}}/#subscribe" target="_blank"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" style="height:1em"><path style="fill:currentColor" d="M502.3 190.8c3.9-3.1 9.7-.2 9.7 4.7V400c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V195.6c0-5 5.7-7.8 9.7-4.7 22.4 17.4 52.1 39.5 154.1 113.6 21.1 15.4 56.7 47.8 92.2 47.6 35.7.3 72-32.8 92.3-47.6 102-74.1 131.6-96.3 154-113.7zM256 320c23.2.4 56.6-29.2 73.4-41.4 132.7-96.3 142.8-104.7 173.4-128.7 5.8-4.5 9.2-11.5 9.2-18.9v-19c0-26.5-21.5-48-48-48H48C21.5 64 0 85.5 0 112v19c0 7.4 3.4 14.3 9.2 18.9 30.6 23.9 40.7 32.4 173.4 128.7 16.8 12.2 50.2 41.8 73.4 41.4z"/></svg> Subscribe</a></li>
```

Comment:
This link should still appear in the header -- I'm honestly not sure how to squeeze it in there in a way that looks good though.